core/src/main/java/org/apache/brooklyn/core/catalog/internal/CatalogBomScanner.java

```diff
@@ -1,130 +0,0 @@
-/*
```

Comment:
Need to remove it from [blueprint.xml](https://github.com/apache/brooklyn-server/blob/3d8675e248cb229e6492d938655cb26222504fbc/karaf/init/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L122) as well.